src/main/java/teammates/common/util/Const.java

```diff
@@ -110,6 +110,8 @@
         TIME_REPRESENTS_DEFAULT_TIMESTAMP = TimeHelper.convertToDate("2011-01-01 00:00 AM UTC");
     }
 
+    public static final String TIME_FORMAT_ISO_8601_UTC = "yyyy-MM-dd'T'HH:mm:ss.S'Z'";
```

Comment:
Apologies I didn't notice this earlier as you'd have to run GodMode again 😅 - could you drop the milliseconds from the timestamp? We don't need that precision and it seems other sites don't use it e.g. GitHub right here (you can inspect the timestamp element of this comment, or any other timestamp on GitHub, to see that they use ISO8601 UTC to carry the timestamp data for formatting in the frontend too, but they don't include milliseconds).